apps/condo/domains/property/schema/CheckPropertyWithAddressExistService.test.js

```diff
@@ -18,14 +20,58 @@ describe('CheckPropertyWithAddressExistService', () => {
         expect(result).toStrictEqual(expectedResult)
     })
     test('user: execute on non-existent address', async () => {
```

Comment:
why mix test and it ?

Reply:
It's also a good practice to write what is expected to be done in test. e.g: user **cant** execute stuff

Reply:
resolved 👍